Ginger/GingerCoreCommon/Run/RunSetConfig.cs

```diff
@@ -56,6 +56,13 @@ public string Name
         /// List of all the agents and the Virtual ones mapped during run 
         /// </summary>
         public List<IAgent> ActiveAgentList = new List<IAgent>();
```

Comment:
Do we need this ActiveAgentList if we have a new list ?

Reply:
no Remove it